README.md

```diff
@@ -24,22 +24,37 @@ mix compile
 iex -S mix
 ```
 
-4. Follow on instructions needed for running the application.
+4. Run the following command to play your first game. Feel free to copy and paste some of the examples provided below to see more shapes!
+Runner.setup(boardSize, aliveCellArray)
```

Comment:
I think idiomatically they're lists in elixir: https://elixir-lang.org/getting-started/keywords-and-maps.html#keyword-lists